bin/kafka-streams-application-reset.sh

```diff
@@ -18,4 +18,4 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
 
-exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.tools.StreamsResetter "$@"
+exec $(dirname $0)/kafka-run-class.sh kafka.tools.StreamsResetter "$@"
```

Comment:
@ijuma @mjsax I would propose to eventual move back to kafka-streams, not kafka-tools to be consistent with admin clients introduced in KIP-4 which may be in kafka-clients, probably also align on the package path as well (for example, if AdminClient will be under o.a.k.clients.tools, then this client could be in o.a.k.streams.tools). But if admin clients will be in kafka-tools then this client would better be in that package as well.